### ISSUE TITLE
combine with superresolution

### ISSUE THREAD

**Issue by Flux9665** [charactr-platform/vocos#33]
Thank you for open sourcing this great work!

One of the great advantages I see in vocoders operating in the time domain is how easy it is to combine the vocoding task with superresolution. You just upsample some more and simply use an audio with a higher samplingrate as the target signal. Is the same somehow possible with Vocos? Could I train a model that uses 16kHz spectrograms as the input but produces a 24kHz wave?

**Comment by lexkoro** [comment]
@Flux9665 Yes it's possible. Just like you said, you would feed 16kHz specs but have 24kHz as target.

**Comment by sankar-mukherjee** [comment]
What setting in the config do i need to change for this?

**Comment by bzp83** [comment]
interested in this as well... did you find out how to do it @sankar-mukherjee ?
